src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -20,16 +24,20 @@ <h2 class="card-pf-title">
     <div class="card-pf-body f8-card-body">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" default-level>
-            <h3>This Space has no Codebases</h3>
-            <div class="f8-blank-slate-main-action">
-              <button id="spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
-            </div>
+          <h3>This Space has no Codebases</h3>
+          <div class="f8-blank-slate-main-action">
+            <ng-container *ngIf="userOwnsSpace">
+              <button id="default-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
+            </ng-container>
+          </div>
         </div>
         <div class="f8-blank-slate-card" *ngIf="codebases.length === 0" user-level>
-            <h3>This Space has no Codebases</h3>
-            <div class="f8-blank-slate-main-action">
-              <button class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
-            </div>
+          <h3>This Space has no Codebases</h3>
+          <div class="f8-blank-slate-main-action">
+            <ng-container *ngIf="userOwnsSpace">
+              <button id="user-level-spacehome-codebases-create-button" class="btn btn-primary btn-lg" (click)="addToSpace.emit()">Add Codebase</button>
```

Comment:
Other than the ID, this button is exactly the same as line 30. I see that the AppLauncher feature has been changed, which makes me question if this should be removed?